megalinter/descriptors/kubernetes.megalinter-descriptor.yml

```diff
@@ -25,6 +25,8 @@ linters:
     install:
       dockerfile:
         - |
-          RUN wget -q https://github.com/instrumenta/kubeval/releases/latest/download/kubeval-linux-amd64.tar.gz \
-              && tar xf kubeval-linux-amd64.tar.gz \
-              && cp kubeval /usr/local/bin
+          RUN MLDL=/downloads/kubeval && mkdir -p ${MLDL} \
+              && wget -P ${MLDL} -q https://github.com/instrumenta/kubeval/releases/latest/download/kubeval-linux-amd64.tar.gz \
+              && tar xf ${MLDL}/kubeval-linux-amd64.tar.gz -C /downloads/kubeval \
```

Comment:
Shouldn't it be `&& tar xf ${MLDL}/kubeval-linux-amd64.tar.gz -C ${MLDL} \` ?

Reply:
Yep..